test-util/src/main/java/io/camunda/zeebe/test/util/record/DecisionRecordStream.java

```diff
@@ -34,4 +34,12 @@ public DecisionRecordStream withDecisionName(final String decisionName) {
   public DecisionRecordStream withDecisionKey(final long decisionKey) {
     return valueFilter(v -> v.getDecisionKey() == decisionKey);
   }
+
+  public DecisionRecordStream withDecisionRequirementsKey(final long decisionRequirementsKey) {
```

Comment:
:thought_balloon: I'm wondering if it would make sense to add a `withDecisionRequirementsId` method. What do you think?

Reply:
This is only used by our tests. I don't see a reason to add an unused method here. We can add it when we need it 🤷